Tests/SwiftParserTest/Assertions.swift

```diff
@@ -594,6 +594,7 @@ func assertParse<S: SyntaxProtocol>(
     )
   } else {
     for (diag, expectedDiag) in zip(diags, expectedDiagnostics) {
+      XCTAssertEqual(diag.fixIts.map { $0.message.message }, expectedDiag.fixIts ?? [], file: file, line: expectedDiag.line)
```

Comment:
We should be checking this exact same thing in
https://github.com/apple/swift-syntax/blob/05356239a2b5b3f5c54653a70ec47cb17d5afd14/Tests/SwiftParserTest/Assertions.swift#L437-L446

Reply:
Changed the logic a bit.